msal/src/main/java/com/microsoft/identity/client/PublicClientApplication.java

```diff
@@ -1110,6 +1079,43 @@ public void acquireTokenSilent(
         }
     }
 
+    @Override
+    public IAuthenticationResult acquireTokenSilent(@NonNull AcquireTokenSilentParameters acquireTokenSilentParameters) throws InterruptedException, MsalException {
+        if (acquireTokenSilentParameters.getCallback() != null) {
+            throw new IllegalArgumentException("Do not provide callback for synchronous methods");
```

Comment:
This is not listed in 'throws'

Reply:
It's a runtime exception... not a checked exception... hence not in Throws.